utest/test/src/test/utest/AssertsTests.scala

```diff
@@ -183,6 +183,11 @@ object AssertsTests extends utest.TestSuite{
         assertMatch(thing){case Seq(1, _, 3) =>}
         ()
       }
+/* Disabled failing tests. I was not able to see what goes wrong since the assertions
+   contain no additional info. The tests started failing after #11679, where
+   we now disallow unapplied functions including synthetic lambdas if the
+   expected type is Unit. I don't now enough about munit to be able to track
```

Comment:
Why is this mentioning `munit`?

Reply:
I haven't looked at these commits in detail. I basically just cherry-picked the commits from Dotty's community build. I would assume that #11679 refers to https://github.com/lampepfl/dotty/pull/11679, but tbh I don't see how it relates to anything that could affect utest (or munit for that matter). Could it be a typo? Maybe @odersky can recall what happened. It might make more sense if we either a) move the test into a Scala 2-only source file, or b) simply remove it altogether, including the comment.